db/fixtures/miq_product_features.yml

```diff
@@ -364,6 +364,10 @@
       :description: Refresh Cloud Providers
       :feature_type: control
       :identifier: ems_cloud_refresh
+    - :name: Re-check Authentication Status 
+      :description: Re-check Authentication Status of Cloud Providers
+      :feature_type: control
+      :identifier: ems_cloud_recheck_auth_status 
```

Comment:
Is this only for ems_cloud types?  I'd think it's for each provider/ems class.  This feature should be more generic or we need one for Ems, Providers, etc.